scripts/config.sh

```diff
@@ -582,7 +582,7 @@ function robo_config_user() {
     groups="$(cat /etc/group | grep ":.*$USER" | grep -o "^[^:]*")"
     for group in $groups_guru; do
         if [ "$(echo $groups | grep "$group")" == "" ]; then
-            sudo usermod "$group" "$USER"
+            sudo usermod -a -G "$group" $USER
```

Comment:
Bitte Variablen immer escapen und bei Parametern diese (meist) in Skripten ausschreiben, damit man weiß was passiert.

```suggestion
            sudo usermod --append --groups "$group" "$USER"
```

Reply:
Bitte die gleichen Anpassungen auf Zeile 577.